mkosi/resources/man/mkosi.1.md

```diff
@@ -1140,16 +1140,17 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     available, with **systemd-sbsign** being preferred.
 
 `Verity=`, `--verity=`
-:   Whether to enforce or disable signed verity for extension images.
-    Takes a boolean value or `auto`. If enabled, a verity key and
-    certificate must be present and the build will fail if we don't
-    detect any verity partitions in the disk image produced by
-    **systemd-repart**. If disabled, verity partitions will be excluded from
-    the extension images produced by **systemd-repart**. If set to `auto` and
-    a verity key and certificate are present, **mkosi** will pass them to systemd-repart
-    and expects the generated disk image to contain verity partitions,
-    but the build won't fail if no verity partitions are found in the
-    disk image produced by **systemd-repart**.
+:   Whether to enforce or disable verity for extension images.
+    Takes a boolean value or one of `hash` and `auto`. If enabled, a verity key
+    and certificate must be present and the build will fail if we don't detect
+    any verity partitions in the disk image produced by **systemd-repart**. If
+    disabled, verity partitions will be excluded from the extension images
+    produced by **systemd-repart**. If set to `hash`, **mkosi** configures
+    **systemd-repart** to create a verity hash partition, but no signature
+    partition. If set to `auto` and a verity key and certificate are present,
+    **mkosi** will pass them to systemd-repart and expects the generated disk
+    image to contain verity partitions, but the build won't fail if no verity
+    partitions are found in the disk image produced by **systemd-repart**.
 
     Note that explicitly disabling signed verity is not yet implemented
```

Comment:
This also needs to mention that `Verity=hash` is not supported for the `disk` output yet.